audit/read_lambda_bucket_policy.tf

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the permissions necessary
+# to read from the bucket containing the COOL lambda deployments.
```

Comment:
Capitalization:
```suggestion
# to read from the bucket containing the COOL Lambda deployments.
```

I'm only flagging it here, you'll have to correct it everywhere.

Reply:
Please see commit [6e282a6](https://github.com/cisagov/cool-accounts/pull/147/commits/6e282a6b7957fc0dbab85bf4b4959705e35e9c23).